model/registry/build.yaml

```diff
@@ -0,0 +1,49 @@
+groups:
+  - id: registry.build
+    prefix: build
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to builds (ci).
+    attributes:
+      - id: artifact.name
+        type: string
+        stability: experimental
+        brief: >
+          The human readable name of the built artifact.
```

Comment:
should it always include the version? Is this just a filename of artifact?

Reply:
Yea, it's often just the filename of the artifact. Release utilities usually add the version, but it not a have to. It's a human readable name & string so anything goes.

Reply:
Understood. Could you please add an example without a version then? Probably 'release-1.tar.gz' is the one, but `1` could be taken as some sort of version. 

We can also add a note that this is a filename of artifact which might include version